CRM/Upgrade/Incremental/MessageTemplates.php

```diff
@@ -280,12 +280,21 @@ protected function getTemplateUpdates() {
       [
         'version' => '5.44.alpha1',
         'upgrade_descriptor' => ts('Use domain and contact tokens instead of smarty values'),
-        'label' => ts('Pledge Transferred'),
+        'label' => ts('Participant Transferred'),
         'templates' => [
           ['name' => 'participant_transferred', 'type' => 'text'],
           ['name' => 'participant_transferred', 'type' => 'html'],
         ],
       ],
+      [
+        'version' => '5.44.alpha1',
+        'upgrade_descriptor' => ts('Use domain and contact tokens instead of smarty values'),
+        'label' => ts('Participant Transferred'),
+        'templates' => [
+          ['name' => 'participant_cancelled', 'type' => 'text'],
```

Comment:
label should be Participant Cancelled?

Reply:
In the message template list it's called "Events - Registration Cancellation Notice", but anyway I think the important part is "cancelled", as opposed to "transferred".

Reply:
Now I'm not even sure where the `label` is used. It shows "Events - Registration Cancellation Notice" on the upgrade screen anyway.

Reply:
Fixed